TwitterAnalysingBot/Twitter_API.py

```diff
@@ -12,29 +12,32 @@
 
 
 date = date.today()
-TickerFilePath = "BotData/TickerScores{Date}.json"
-
+TickerScoresFilePath = "BotData/TickerScores{Date}.json"
+CandidateTickers = r'..\CandidateTickers.txt'
+MaxCandidateSizeBytes = 1663
+Tickers = {}
 
 #Make a list of all the ticker symbols in the 'MoreTickers' file
-with open(r'TwitterAnalysingBot\MoreTickers.txt', 'r') as f:
-        lines = f.readlines(1663)
-    
-        lines = [x.strip() for x in lines]
+with open(CandidateTickers, 'r') as f:
+    lines = f.readlines(MaxCandidateSizeBytes)
+    #Create a python dictionary with the reset hypescores of all the stocks
+    Tickers = dict.fromkeys([x.strip() for x in lines], 0)
 
-#Create a python dictionary with the reset hypescores of all the stocks
-Tickers = {}
-for Ticker in lines:
-    Tickers[Ticker] = 0
+class StdOutListener(StreamListener):
 
+    def __init__(self, configurationFilePath):
+        #Load configuration file in memory once with the constructor.
+        with open('Configuration.json', 'r') as f:
```

Comment:
Could I replace the "Configuration.json" file path with the one I input in the constructor?